src/tabset/tabset.ts

```diff
@@ -91,40 +90,36 @@ export interface NgbTabChangeEvent {
   preventDefault: () => void;
 }
 
+
 /**
  * A component that makes it easy to create tabbed interface.
  */
 @Component({
   selector: 'ngb-tabset',
   exportAs: 'ngbTabset',
   template: `
-    <ul [class]="'nav nav-' + type + (orientation == 'horizontal'?  ' ' + justifyClass : ' flex-column')" role="tablist">
-      <li class="nav-item" *ngFor="let tab of tabs">
-        <a [id]="tab.id" class="nav-link" [class.active]="tab.id === activeId" [class.disabled]="tab.disabled"
-          href (click)="select(tab.id); $event.preventDefault()" role="tab" [attr.tabindex]="(tab.disabled ? '-1': undefined)"
-          [attr.aria-controls]="(!destroyOnHide || tab.id === activeId ? tab.id + '-panel' : null)"
-          [attr.aria-selected]="tab.id === activeId" [attr.aria-disabled]="tab.disabled">
-          {{tab.title}}<ng-template [ngTemplateOutlet]="tab.titleTpl?.templateRef"></ng-template>
-        </a>
+    <ul ngbNav
+        (navChange)="tabChange.next($event)"
+        [destroyOnHide]="destroyOnHide"
+        [activeId]="activeId"
+        class="nav nav-{{type}} {{orientation == 'horizontal' ?  '' + justifyClass : 'flex-column'}}"
```

Comment:
if the `tablist` role is added, the `vertical` orientation reuqires the `aria-orientation` here:
https://www.w3.org/TR/wai-aria-1.1/#tablist
e.g. https://github.com/twbs/bootstrap/pull/28927/commits/8de655840856a5f10ee83b9028fc593a80d40180

Reply:
Good catch, thanks for the review. 
Seems more like an exisitng tabset bug, will fix it separately.

However that made me thinking that we should probably have the `[orientation]` input on the `ngbNav` to set it automatically.